.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+Fixes #
+
+# Description
+<!---
+Thanks for opening a PR!
+
+Please add your description here of changes made and how they are going to
+resolve the linked issue 
+-->
+
+# Checklist
+<!---
+Before requesting review, please go through this checklist:
+-->
+
+- [ ] Links to correct issue
+- [ ] Update changelog, if changes affect users
+- [ ] PR title starts with ``#issue-nr``, e.g. #737
```

Comment:
We want to have in the form `Issue #nr`
e.g. `Issue #737`

Reply:
Fixed